main/managementinterface.go

```diff
@@ -218,6 +218,12 @@ func handleSettingsSetRequest(w http.ResponseWriter, r *http.Request) {
 					case "Ping_Enabled":
 						globalSettings.Ping_Enabled = val.(bool)
 					case "GPS_Enabled":
+						switch {
+						case val.(bool) == true && globalSettings.GPS_Enabled == false:
+							connectGpsd(globalSettings.GpsdAddress)
```

Comment:
We could make the simplifying assumption that gpsd should always be running and always connect, but perhaps not use the data if `GPS_Enabled` is `false`.